BedrockCommand.cpp

```diff
@@ -27,9 +29,12 @@ BedrockCommand::~BedrockCommand() {
     for (auto request : httpsRequests) {
         request->owner.closeTransaction(request);
     }
+    if (countCommand) {
+        _commandCount--;
+    }
 }
 
-BedrockCommand::BedrockCommand(SQLiteCommand&& from) :
+BedrockCommand::BedrockCommand(SQLiteCommand&& from, int dontCount) :
```

Comment:
why not a bool for `dontCount`?

Reply:
What I actually want here is something like: http://www.cplusplus.com/reference/mutex/defer_lock/

But this was easier. I like that solution because you can only pass something explicit, like `BedrockCommand::COUNT` or `BedrockCommand::DONT_COUNT` instead a bool or int that's pretty arbitrary looking.

So if I change it I'll probably do that.